Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -205,6 +206,7 @@ def bottle_formula(f)
         tab.poured_from_bottle = false
         tab.HEAD = nil
         tab.time = nil
+        tab.changed_files = changed_files
```

Comment:
Here's my `INSTALL_RECEIPT.json` for `mysql`:

```json
{
  "used_options": [],
  "unused_options": [
    "--with-test",
    "--with-embedded",
    "--with-archive-storage-engine",
    "--with-blackhole-storage-engine",
    "--with-local-infile",
    "--with-debug"
  ],
  "built_as_bottle": true,
  "poured_from_bottle": true,
  "changed_files": [
    "INSTALL_RECEIPT.json",
    "bin/mysql_config",
    "bin/mysqld_multi",
    "bin/mysqld_safe",
    "homebrew.mxcl.mysql.plist",
    "include/mysql/my_config.h",
    "lib/pkgconfig/mysqlclient.pc",
    "mysql-test/Makefile",
    "mysql-test/suite/ndb/include/have_java.inc",
    "share/doc/mysql/INFO_BIN",
    "share/man/man1/myisam_ftdump.1",
    "share/man/man1/mysql.server.1",
    "share/man/man1/mysql_config.1",
    "share/man/man1/mysql_install_db.1",
    "share/man/man1/mysqld_multi.1",
    "share/man/man1/mysqld_safe.1",
    "share/man/man1/mysqldumpslow.1",
    "support-files/mysql-log-rotate",
    "support-files/mysql.server",
    "support-files/mysqld_multi.server"
  ],
  "time": 1476101424,
  "source_modified_time": 1472126415,
  "HEAD": null,
  "stdlib": "libcxx",
  "compiler": "clang",
  "runtime_dependencies": [],
  "source": {
    "path": "/usr/local/Library/Taps/homebrew/homebrew-core/Formula/mysql.rb",
    "tap": "homebrew/core",
    "spec": "stable",
    "versions": {
      "stable": "5.7.15",
      "devel": null,
      "head": null,
      "version_scheme": 0
    }
  }
}
```

Not too bad, I think. Would you rather output to a separate file?